assets/pages/oneoff-contributions/components/formFields.jsx

```diff
@@ -74,17 +76,19 @@ function NameForm(props: PropTypes) {
 
   return (
     <form className="oneoff-contrib__name-form">
-      <TextInput
-        id="email"
-        value={props.email.value}
-        labelText="Email"
-        placeholder="Email"
-        onChange={props.setEmail}
-        onBlur={onFormFieldBlur(props.setEmailShouldValidate, stripeButtonClassName)}
-        modifierClasses={['email']}
-        showError={shouldShowError(props.email)}
-        errorMessage="Please enter a valid email address."
-      />
+      { !props.isSignedIn ? (
```

Comment:
not sure what existing precedent there is in the codebase... maybe this?

```javascript
{!props.isSignedIn ? (
  <TextInput
    id="email"
    value={props.email.value}
    labelText="Email"
    placeholder="Email"
    onChange={props.setEmail}
    onBlur={onFormFieldBlur(props.setEmailShouldValidate, stripeButtonClassName)}
    modifierClasses={['email']}
    showError={shouldShowError(props.email)}
    errorMessage="Please enter a valid email address."
  />) : null}
```

or this??

```javascript
{
  !props.isSignedIn ? (
    <TextInput
      id="email"
      value={props.email.value}
      labelText="Email"
      placeholder="Email"
      onChange={props.setEmail}
      onBlur={onFormFieldBlur(props.setEmailShouldValidate, stripeButtonClassName)}
      modifierClasses={['email']}
      showError={shouldShowError(props.email)}
      errorMessage="Please enter a valid email address."
    />
  ) : null
}
```